### DISCUSSION TITLE
求教，签到任务单独步骤测试全正常，整体测试时有http 500报错

### DISCUSSION THREAD

**Discussion by Nigel-NI** [qd-today/qd#432] [Q&A] [answered]
任务流程：
1，进入登录界面获取网站提供的xsrf-token；
2，xsrf-token转码；
3，发送含有用户名、密码、xsrf-token的请求模拟点击登录；
...

这三个步骤单独、依次测试时是可以正常完成登录的，第三步结果会正常返回账号信息；但整体测试时会报错，在流程第三步时提示：

> HTTP 500: Internal Server Error

经测试发现这个错误无论用户名、密码正确与否都会出现。请问这种情况有什么解决办法吗？

**Comment by a76yyyy** [comment, marked as answer]
你是不是通过 header 正则提取 xsrf-token, 由于header内容前后端的正则不一致导致前端单个测试正常, 但整体测试出错

这个bug在最新源码里修复了, 请更新至最新源码或者使用 dev 标签的容器进行尝试, 

如果还有问题, 请在容器环境变量开启 Debug 模式, 在容器的后端日志中获取详细的错误信息.

**Reply by Nigel-NI** [reply]
感谢回复，docker切换到dev版问题就不再出现了
